src/fiatconnect/saga.ts

```diff
@@ -67,6 +76,23 @@ export function* watchFetchFiatConnectQuotes() {
   yield takeLeading(fetchFiatConnectQuotes.type, handleFetchFiatConnectQuotes)
 }
 
+export function* handleFetchFiatConnectProviders() {
+  const account: string = yield select(currentAccountSelector)
+  try {
+    if (!account) {
```

Comment:
Not directly related to this PR, but I'm forgetting why we require an account address to return eligible providers. Looking at the [implementation](https://github.com/valora-inc/valora-rest-api/blob/main/src/fiatConnect/getFiatConnectProviders.ts), it looks like it's only used to allow whitelisting for specific users before a provider is actually "live". Not for this PR, but it seems like we could make this an optional param and avoid a failure case here.

Reply:
Yea I think that's a good point since its not necessary for normal production use